resolwe_bio/processes/workflows/chip_seq.yml

```diff
@@ -357,3 +357,341 @@
+            uses linear scaling. This option will make the results unstable and irreproducible
+            since each time, random reads would be selected, especially the numbers (pileup,
+            pvalue, qvalue) would change. Consider to use 'randsample' script before MACS2 runs
+            instead.
+        - name: bedgraph
+          label: Save fragment pileup and control lambda
+          type: basic:boolean
+          default: true
+          description: |
+            If this flag is on, MACS will store the fragment pileup, control lambda, -log10pvalue
+            and -log10qvalue scores in bedGraph files. The bedGraph files will be stored in
+            current directory named NAME+'_treat_pileup.bdg' for treatment data,
+            NAME+'_control_lambda.bdg' for local lambda values from control,
+            NAME+'_treat_pvalue.bdg' for Poisson pvalue scores (in -log10(pvalue) form), and
+            NAME+'_treat_qvalue.bdg' for q-value scores from Benjamini-Hochberg-Yekutieli
+            procedure.
+        - name: spmr
+          label: Save signal per million reads for fragment pileup profiles
+          type: basic:boolean
+          default: true
+          disabled: "settings.bedgraph === false"
+        - name: call_summits
+          label: Call summits
+          type: basic:boolean
+          default: false
+          description: |
+            MACS will now reanalyze the shape of signal profile (p or q-score depending on cutoff
+            setting) to deconvolve subpeaks within each peak called from general procedure. It's
+            highly recommended to detect adjacent binding events. While used, the output subpeaks
+            of a big peak region will have the same peak boundaries, and different scores and peak
+            summit positions.
+        - name: broad
+          label: Composite broad regions
+          type: basic:boolean
+          default: false
+          disabled: "settings.call_summits === true"
+          description: |
+            When this flag is on, MACS will try to composite broad regions in BED12 (a
+            gene-model-like format) by putting nearby highly enriched regions into a broad region
+            with loose cutoff. The broad region is controlled by another cutoff through
+            --broad-cutoff. The maximum length of broad region length is 4 times of d from MACS.
+        - name: broad_cutoff
+          label: Broad cutoff
+          type: basic:decimal
+          required: false
+          disabled: "settings.call_summits === true || settings.broad !== true"
+          description: |
+            Cutoff for broad region. This option is not available unless --broad is set. If -p is
+            set, this is a p-value cutoff, otherwise, it's a q-value cutoff. DEFAULT = 0.1
+  run:
+    language: workflow
+    program:
+      - id: macs2
+        run: macs2-callpeak
+        input:
+          case: '{{input.case}}'
```

Comment:
In some other workflows, we use the syntax `{{ input.case }}` (space between contents and brackets).